.travis.yml

```diff
@@ -1,37 +1,38 @@
-language: go
-
 addons:
   hosts:
     - quic.clemente.io
 
-go:
-  - 1.x
-  - tip
-
 matrix:
-  allow_failures:
-    - go: tip
-  fast_finish: true
+  include:
+    - language: go
+      go: 1.11.x
+      env: GO111MODULE=on
+    - language: go
+      go: 1.x
+      env: GO111MODULE=auto
+    - language: go
+      go: tip
+      env: GO111MODULE=auto
+
+allow_failures:
+  - go: tip
+fast_finish: true
 
 before_install:
   # Decrypts a script that installs an authenticated cookie
   # for git to use when cloning from googlesource.com.
   # Bypasses "bandwidth limit exceeded" errors.
   # See github.com/golang/go/issues/12933
- - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_3df18f9af81d_key -iv $encrypted_3df18f9af81d_iv -in dist/gitcookie.sh.enc -out dist/gitcookie.sh -d; fi
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_3df18f9af81d_key -iv $encrypted_3df18f9af81d_iv -in dist/gitcookie.sh.enc -out dist/gitcookie.sh -d; fi
+  - go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
https://github.com/golangci/golangci-lint/blob/master/README.md#ci-installation

Reply:
That's how I had it originally but the everything else was `go get` so I switched it.  I'll update it to that.  Thx.